ichnaea/data/tests/test_export.py

```diff
@@ -558,16 +579,18 @@ def test_cell_duplicated(self, celery, session):
         assert len(cells) == 1
         assert cells[0].samples == 1
 
-    def test_cell_invalid(self, celery, session, stats):
+    def test_cell_invalid(self, celery, session, metricsmock):
         self.add_reports(celery, cell_factor=1, wifi_factor=0, cell_mcc=-2)
         self._update_all(session)
 
-        stats.check(
-            counter=[
-                ("data.report.upload", 1, 1, ["key:test"]),
-                ("data.report.drop", 1, 1, ["key:test"]),
-                ("data.observation.drop", 1, 1, ["type:cell", "key:test"]),
-            ]
+        assert metricsmock.has_record(
+            "incr", "data.report.upload", value=1, tags=["key:test"]
+        )
+        assert metricsmock.has_record(
+            "incr", "data.report.drop", value=1, tags=["key:test"]
+        )
+        assert metricsmock.has_record(
+            "incr", "data.observation.drop", value=1, tags=["type:cell", "key:test"]
```

Comment:
These don't assert that there's only one of each of these records. I'll fix that.